plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftAutomaticJoinPushdown.java

```diff
@@ -38,6 +39,7 @@ protected QueryRunner createQueryRunner()
                 ImmutableList.of());
     }
 
+    @Test
     @Override
     @Disabled
```

Comment:
If this test is disable what's the purpose of @Test annotation?

Reply:
This is to make `io.airlift.junit.ReportMissingTestAnnotation.reportMissingTestAnnotations` be happy - or else it would throw exception. I'm not sure if there is a different way to solve this issue